.github/workflows/muzzle.yml

```diff
@@ -41,16 +26,8 @@ jobs:
           distribution: temurin
           java-version: 11
 
-      # don't use gradle-build-action because all the parallel muzzle jobs cause the cache service
-      # to respond with 429 (to both muzzle and other jobs running in parallel)
-
       - name: Run muzzle
-        # using retry because of sporadic gradle download failures
-        uses: nick-invision/retry@v2.6.0
+        uses: gradle/gradle-build-action@v2
         with:
-          # timing out has not been a problem, these jobs typically finish in 2-3 minutes
-          timeout_minutes: 15
-          # give maven central some time to hopefully recover (120 seconds has not proven to be enough)
-          retry_wait_seconds: 300
-          max_attempts: 5
-          command: ./gradlew ${{ matrix.module }}:muzzle
+          arguments: ${{ matrix.task }} --max-workers=4
```

Comment:
Can we try the split jobs without max-workers?

Reply:
```suggestion
          arguments: ${{ matrix.task }}
```

Reply:
no real difference 👍